src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
BTW it might be a good idea to add either a `.vscode\settings.json` where we can define `"files.trimTrailingWhitespace": false` and other desired settings.  Or we could cast a wider net and use a top-level `.editorconfig` file with `trim_trailing_whitespace = true`.  In this case, I would add a `.vscode\extensions.json` file with the EditorConfig extension listed.  In fact, such a file should also recommend the C#, C++ and PowerShell extensions from Microsoft.